### ISSUE TITLE
Accessibility: Complianz "Subtitles" should be heading 3s

### ISSUE THREAD

**Issue by mrwweb** [rlankhorst/complianz-gdpr#470]
Paragraphs with the class "cmplz-subtitle" such as on the Opt-out Preferences Document should be Heading 3 (`h3`) elements. They are clearly styled and function as such, so should be marked up to match.

**Comment by mrwweb** [comment]
Pleased to see this got resolved in Premium version 7.5.7.3. Hope that change makes it into the free version soon :)
